koku/api/provider/provider_manager.py

```diff
@@ -107,6 +107,27 @@ def get_previous_month_data_exists(self):
             completed_datetime__isnull=False,
         ).exists()
 
+    def get_state(self):
+        """Get latest manifest state."""
+        states = {
+            "download": "pending",
+            "processing": "pending",
+            "summary": "pending",
+        }
```

Comment:
Can we take advantage of the `ManifestStep` enum here since those seem to be the same options?

Reply:
Yeh we could change this